cg/server/api.py

```diff
@@ -363,23 +359,25 @@ def parse_pool(pool_id):
 
 
 @BLUEPRINT.route("/flowcells")
-def parse_flow_cells() -> Any:
-    """Return flow cells."""
-    flow_cells: list[Flowcell] = db.get_flow_cell_by_name_pattern_and_status(
-        flow_cell_statuses=[request.args.get("status")],
-        name_pattern=request.args.get("enquiry"),
+def parse_sequencing_runs() -> Any:
+    """Return Illumina sequencing runs."""
+    sequencing_runs: list[IlluminaSequencingRun] = (
+        db.get_illumina_sequencing_runs_by_data_availability_and_id_pattern(
+            data_availability=[request.args.get("status")],
+            pattern=request.args.get("enquiry"),
+        )
     )
-    parsed_flow_cells: list[dict] = [flow_cell.to_dict() for flow_cell in flow_cells[:50]]
-    return jsonify(flowcells=parsed_flow_cells, total=len(flow_cells))
+    parsed_runs: list[dict] = [run.to_dict() for run in sequencing_runs[:50]]
+    return jsonify(flowcells=parsed_runs, total=len(sequencing_runs))
```

Comment:
we need to make sure that the other end can handle the data from the new dict. 

@Vince-janv you mentioned you know where this is used? Can you point us there? I think it would be good to setup a DTO and return that with the required information.

Reply:
Wiat for update: if not used deprecate

Reply:
Update: We are droping the endpoints. Resolving